src/Sulu/Component/Content/Repository/ContentRepository.php

```diff
@@ -583,13 +580,10 @@ private function resolveContent(
         Row $row,
         $locale,
         $locales,
-        $webspaceKey,
```

Comment:
Webspace-Key can be extracted from the path. this avoids some problems when using multiple webspaces in the same query.